package.json

```diff
@@ -24,37 +21,37 @@
     "server": "laboperator-middleware server"
   },
   "dependencies": {
-    "ajv": "^6.12.6",
-    "camelcase-keys": "^6.2.2",
-    "express": "^4.17.3",
-    "fetch-retry": "^4.0.1",
-    "form-data": "^3.0.0",
-    "fs-extra": "^9.0.1",
-    "http-errors": "^1.8.0",
-    "http-status-codes": "^2.1.4",
-    "https-proxy-agent": "^5.0.0",
+    "ajv": "^8.12.0",
+    "ajv-formats": "^2.1.1",
+    "camelcase-keys": "^8.0.2",
+    "express": "^4.18.2",
+    "fetch-retry": "^5.0.4",
+    "form-data": "^4.0.0",
+    "fs-extra": "^11.1.0",
+    "http-errors": "^2.0.0",
+    "http-status-codes": "^2.2.0",
+    "https-proxy-agent": "^5.0.1",
     "lodash": "^4.17.21",
     "morgan": "^1.10.0",
-    "nodemon": "^2.0.6",
-    "snakecase-keys": "^3.2.1",
-    "swagger-client": "^3.12.1",
-    "winston": "^3.3.3",
-    "yaml": "^1.10.0",
-    "yargs": "^16.2.0"
+    "nodemon": "^2.0.20",
+    "snakecase-keys": "^5.4.5",
+    "swagger-client": "^3.18.5",
+    "winston": "^3.8.2",
+    "yaml": "^2.2.1",
+    "yargs": "^17.7.1"
   },
   "devDependencies": {
-    "chai": "^4.2.0",
-    "chai-http": "^4.3.0",
-    "chai-subset": "^1.6.0",
-    "dirty-chai": "^2.0.1",
+    "@types/node": "^18.14.4",
+    "@types/supertest": "^2.0.12",
+    "eslint": "^8.35.0",
     "fetch-mock": "^9.11.0",
-    "laboperator-middleware-development": "https://github.com/labforward/laboperator-middleware-development",
-    "sinon": "^9.2.2",
-    "sinon-chai": "^3.5.0"
+    "laboperator-middleware-development": "labforward/laboperator-middleware-development#8ccb2cb043b6d542ce6b30b81f8c9df6407c4b31",
```

Comment:
Are we purposefully sticking to this commit or did you forget to remove it? 😅

Reply:
will be updated, it has to be pinned to a specific commit until the PR in laboperator-middleware-development is merged..